lib/currency.js

```diff
@@ -1,22 +1,38 @@
-const { Amount, pkg: hsdPkg } = require('hsd');
+const { Amount } = require('hsd');
+
+const pkgs = {
+  handshake: require('hsd/lib/pkg'),
+  bitcoin: require('bcoin/lib/pkg'),
+};
```

Comment:
we could probably write a function to generate the `CURRENCY_UNITS` object using `pkgs`